contrib/sssd.spec.in

```diff
@@ -612,6 +613,8 @@ cp $RPM_BUILD_ROOT/%{_datadir}/sssd/krb5-snippets/sssd_enable_idp \
 %if %{build_passkey}
 cp $RPM_BUILD_ROOT/%{_datadir}/sssd/krb5-snippets/sssd_enable_passkey \
    $RPM_BUILD_ROOT/%{_sysconfdir}/krb5.conf.d/sssd_enable_passkey
+install -D -m 0644 contrib/90-sssd-token-access.rules %{buildroot}%{_udevrulesdir}/90-sssd-token-access.rules
+sed -i 's@:sssd:@:%{sssd_user}:@g' %{buildroot}%{_udevrulesdir}/90-sssd-token-access.rules
```

Comment:
I think it's better to skip `install` altogether if non-root support isn't built (i.e. %{sssd_user}==root)

We still need to put value of '--with-sssd-user=' into rule otherwise.
This is typically done during build, not in the spec file, like:
https://github.com/SSSD/sssd/blob/9d1fccb5e9c83a734d2fd0b4d50e19684bb94ae3/src/sysv/systemd/sssd-pac.service.in#L17
https://github.com/SSSD/sssd/blob/9d1fccb5e9c83a734d2fd0b4d50e19684bb94ae3/Makefile.am#L5269

Not sure this is the best approach, but at least for the consistency...

Reply:
I ended up generating the file through `configure`.